src/components/TextField/TextField.stories.mdx

```diff
@@ -69,27 +70,67 @@ Regular TextField with placeholder
   </Story>
 </Preview>
 
-# Text Field with Placeholder and icon
+# Text Field with Placeholder, icon and size
 
-Regular TextField with placeholder and icon
+Regular TextField with placeholder, icon and size
 
 <Preview>
-  <Story name="Text Field with Placeholder and icon">
+  <Story name="Text Field with Placeholder, icon and size">
     <Stack>
       <TextField
+        size={select('size', ['md','sm'],'sm')}
         lean={boolean('lean version', false)}
         leftIcon={
           <img src="https://brandmark.io/logo-rank/random/pepsi.png" width="16" height="16" />
         }
         placeholder="placeholder"
         styleType={select('style', ['filled', 'outlined', 'elevated'], 'filled')}
       />
+      <TextField
+        size={select('size', ['md','sm'], 'sm')}
+        lean={boolean('lean version', false)}
+        rightIcon={
+          <img style={{background: 'red'}} src="https://brandmark.io/logo-rank/random/pepsi.png" width="16" height="16" />
+        }
+        placeholder="placeholder"
+        styleType={select('style', ['filled', 'outlined', 'elevated'], 'filled')}
+      />
+      <TextField
+        size={select('size', ['md','sm'], 'sm')}
+        lean={boolean('lean version', false)}
+        label={'Label'}
+        leftIcon={
+          <img style={{background: 'red'}} src="https://brandmark.io/logo-rank/random/pepsi.png" width="16" height="16" />
+        }
+        styleType={select('style', ['filled', 'outlined', 'elevated'], 'filled')}
+      />
     </Stack>
   </Story>
 </Preview>
 
-# TextField disabled with label
 
+# Text Field with color type and icon picker
+
+Regular TextField with color type and icon picker
+
+<Preview>
+  <Story name="Text Field with color type and icon picker">
+    <Stack>
+      <TextField
+        fill={select('fill', flatColors, 'lightGray')}
+        fillShade={select('fillShade', colorShades, 100)}
+        size={select('size', ['md','sm'], 'sm')}
+        lean={boolean('lean version', false)}
+        label={'Label'}
+        LeftIconWithSwatches={({colorConfig}) =>
+          <Icon name={select('name',Object.keys(iconSelector),'search')} colorConfig={colorConfig} />}
```

Comment:
the `name` here is confusing, can you change it to `leftIconName` or `IconName`?

Reply:
Maybe we should change the name of the existing "leftIcon" to "leftImage" and rename this to leftIcon but I have to first find the breaking changes inside the storybook components.